### PR TITLE
Template system

### PR DESCRIPTION
**Pull request by mikitachab**
- added template system setup
- added Bootstrap setup
- added bootstrap navbar
- added players list view

solution of: #17, #16, #6

### CODE REVIEW

File: template/navbar.html [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
-<h1>Navigation Bar</h1>
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <a class="navbar-brand" href="#">Score Board</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav">
+        <li class="nav-item active">
+          <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
```
Comment (antonskwr) [inline template/navbar.html]: Whitespace after "Home"

Comment (mikitachab) [reply template/navbar.html]: idk :), this is copy pasted from https://getbootstrap.com/docs/4.0/components/navbar/#nav
i can fix it if you insist :)

Comment (antonskwr) [reply template/navbar.html]: Let's remove it, otherwise we should find an explanation why the whitespace is there.

Comment (mikitachab) [reply template/navbar.html]: done

**Review by antonskwr — APPROVED** [review]

**Pull request merged**